src/font.md

```diff
@@ -3,3 +3,5 @@ ___
 The preferred family font for the project would be JetBrains Mono, but if not available we could also use Cascadia Code
 
 Did some changes just to that it seems that it attended the review xd
+
+But again, I'm adding something.
```

Comment:
Maybe its not necessary, you should add some other libraries.

Reply:
Looks good!!!